### ISSUE TITLE
Redis ttl and pttl returns None when ttl value is 0

### ISSUE THREAD

**Issue by marakal** [andymccurdy/redis-py#693]
``` python
r=redis.Redis()
print r.set("foo", "bar")
print r.expire("foo", 1)
time.sleep(0.5)
print r.ttl("foo")
print r.get("foo")
```

prints

```
True
True
None
bar
```

but ttl should return 0 in this case, expire is set

**Comment by AndreaCrotti** [comment]
+1 overall it seems to behave weirdly, returning None in both the cases where redis would return -1 or -2..

**Comment by jkasarherou** [comment]
it works correctly if you use
`r = redis.StrictRedis()`

**Comment by micahculpepper** [comment]
👍  for this issue. Per the command reference at https://redis.io/commands/ttl, ttl should return a `-1` when expire is not set, and this is exactly what I see when hitting my redis instance directly. When I try to check the ttl on a key that does not have an expire set using this library, I get `None`.

**Comment by gyermolenko** [comment]
Tested with python3.7.3, redis3.2.1 - not reproduced. 

```python3
# x.py
import redis
import time
r=redis.Redis()

r.set("exp", "expval")
r.expire("exp", 1)
time.sleep(0.5)

print(r.ttl("exp"))
print(r.pttl("exp"))
print(r.get("exp"))
time.sleep(1)

print(r.ttl("exp"))
print(r.pttl("exp"))
print(r.get("exp"))
```

```bash
$ python x.py 
0
499
b'expval'
-2
-2
None
```

**Comment by andymccurdy** [comment]
This was resolved as part of redis-py 3.0. Closing this.
